docs/user_guide/HOWTO/Using_the_SIMP_Release_RPM.rst

```diff
@@ -0,0 +1,126 @@
+.. _howto-use-the-simp-release-rpm:
```

Comment:
I ran through this real quick and, while it is inferred, we should give people an explicit link to the installation guide at this point. Otherwise, they may think that they have a functional product and be confused.

Reply:
I could be wrong, but I believe this line is creating a reference for itself to be called elsewhere, not calling a separate file. Unless I'm misunderstanding what you mean.